test/e2e/components/vl-map.js

```diff
@@ -129,6 +125,29 @@ class VlMap extends VlElement {
   async _getToggleFullscreenButton() {
     return this.shadowRoot.findElement(By.css('.ol-full-screen'));
   }
+
+  async _moveToPointOnMap(pixel) {
+    const rect = await this.getRect();
+    return {
+      origin: this,
+      x: Math.round(pixel[0] - (rect.width / 2)),
+      y: Math.round(pixel[1] - (rect.height / 2)),
+      duration: 1000,
+    };
+  }
+
+  async sleepRechthoek(coordinatesStart, coordinatesEnd) {
```

Comment:
Engels ipv nederlands?

Reply:
ja tom wil engels in webcomponenten denk ik ja